src/main/interface/cli.c

```diff
@@ -2221,6 +2224,49 @@ static void cliName(char *cmdline)
     printName(DUMP_MASTER, pilotConfig());
 }
 
+#define ERROR_MESSAGE "Error, %s is already set: %s"
+
+static void cliBoardName(char *cmdline)
+{
+    const unsigned int len = strlen(cmdline);
+    if (len > 0 && boardInformationIsSet() && strncmp(getBoardName(), cmdline, len)) {
+        cliPrintLinef(ERROR_MESSAGE, "board_name", getBoardName());
+    } else {
+        if (len > 0) {
+            setBoardName(cmdline);
+            boardInformationUpdated = true;
```

Comment:
this is set when `boardInformationIsSet` and name did not change

Reply:
Yes. Which does not matter, since the `persistBoardInformation()` call does not do anything if `boardInformationIsSet()`.